### DISCUSSION TITLE
230630 팀 그라운드 룰 회의 ( 11:40 ~ 12:30 )

### DISCUSSION THREAD

**Discussion by splitCoding** [woowacourse-teams/2023-shook#3] [회의]
## 기존의 팀 그라운드를 재검토한다.

**Comment by Cyma-s** [comment]
### 👥 관계

- 상호 존중하는 수평적인 관계 유지하기
(편하게 말을 하되 다만 선넘지 않게 하기)
- 프로젝트는 코어 타임에 진행하기
    - 코어 타임은 **화-금 오전 10시~ 오후 6시 30분, 월 오후 1시 ~ 오후 6시 30분** 이다.
    - 코어 타임 이후 생기는 이슈들은 Github Issue에 등록한다.
- 시시콜콜한 이야기도 많이 하는 사이
- 의견이 있으면 무조건 말하기
    - 의견을 제시할 때에는 근거를 충분하게 제시한다.
    - “아니 근데”, “당연히 ~~ 아니지” 와 같은 말은 금지한다.
    - 다른 사람이 말하는 도중에 하고 싶은 말이 생기면 손을 들어 발언권을 얻는다.
- Don't ask to ask
    - 바보같은 질문은 없다. 잘 몰라도 질문하자.
- 시간을 선택하는 것은 시간을 절약하는 것
- 팀의 의견은 모두의 의견

### 스크럼

- 데일리 미팅 (10시 ~ 10시 30분)
        - 잡담만 한다.
        - 데일리 미팅에 지각하면 전체 커피를 쏜다.
- 데일리 미팅 이후, 어제 발생한 이슈를 공유한다.
- 데일리 스크럼 (18시 ~ 18시 30분)
        - 자신이 한 일을 공유한다.
        - 어려웠던 점과 같이 해결하고 싶은 것을 공유한다.
        - 내일 할 일을 정한다.
- 데일리 스크럼 내용은 서기가 기록한다.
- 월요일에 주 단위 스프린트에 대해 이야기한다. (2시간~3시간)
    - 스프린트 기간은 1주로 정한다.

### 회고

- 감정 회고
    - 매주 금요일 데일리 스크럼 때 자유로운 분위기에서 Emotion-Meeting 을 진행한다.
    - 감정 회고는 두괄식으로 진행한다. Ex. 나는 ~~가 좋았다/싫었다
    - 솔직하게 말한다.
- 스프린트 회고
    - 각 파트별로 진행한다.
    - 매주 월요일 날 이전 스프린트에 대한 회고를 진행한다.
    - 스프린트 회고 내용은 Wiki에 작성한다.

### 🤝회의

- 회의는 반드시 1시간이 넘지 않도록 한다.
- 매 회의가 끝나면 쉬는 시간을 갖는다.
- 회의 안건을 제시한 사람이 예상 회의 시간을 공지한다.
- 회의의 방향을 주기적으로 체크한다.
- 서기는 회의 내용을 Discussion에 남긴다.
- 회의는 미리 정하고 진행한다. 
    - 전 날에 Discussion에 등록한다.

### 지식 공유

- 나만 알거나, 나만 모르는 지식이 없어야 한다.
- 해결한 문제에 대해서 Wiki에 글로 공유한다.
- 필요하면 파트별로 스터디를 자유롭게 진행한다.
- 파트끼리 공유할 공통 지식이 있다면 적극적으로 공유한다.

**Comment by splitCoding** [comment]
13:30 팀 게임 일정 이후 추가로 진행한다.
